ember-cli-build.js

```diff
@@ -0,0 +1,20 @@
+/* global require, module */
+var EmberApp = require('ember-cli/lib/broccoli/ember-addon');
+
+module.exports = function(defaults) {
+  var app = new EmberApp(defaults, {
+    // See https://github.com/ember-cli/ember-cli/issues/4084
+    sassOptions: {
+      extension: 'scss'
```

Comment:
This is currently required due to a bug https://github.com/ember-cli/ember-cli/issues/4084